src/main/resources/application.yaml

```diff
@@ -165,6 +158,7 @@ refresh:
     pageSize: ${REFRESH_JOB_PAGE_SIZE:400}
     sortDirection: ${REFRESH_JOB_SORT_DIR:ASC}
     sortColumn: ${REFRESH_JOB_SORT_COL:}
+  authorisedServices: am_org_role_mapping_service,am_role_assignment_refresh_batch
```

Comment:
As there are two refresh APIs can we indent this so it is falls under the Refresh.Job config, so it is clear that this only applies to the Refresh.job API?